doc/man3/OSSL_trace_set_channel.pod

```diff
@@ -128,59 +128,59 @@ prematurely.
 A suggestion is to make such cleanup part of a function that's
 registered very early with L<atexit(3)>.
 
-=item C<OSSL_TRACE_CATEGORY_TLS>
+=item B<OSSL_TRACE_CATEGORY_TLS>
 
 Traces the TLS/SSL protocol.
 
-=item C<OSSL_TRACE_CATEGORY_TLS_CIPHER>
+=item B<OSSL_TRACE_CATEGORY_TLS_CIPHER>
 
 Traces the ciphers used by the TLS/SSL protocol.
 
-=item C<OSSL_TRACE_CATEGORY_ENGINE_TABLE>
+=item B<OSSL_TRACE_CATEGORY_ENGINE_TABLE>
 
 Traces the ENGINE algorithm table selection.
 
 More precisely, engine_table_select(), the function that is used by
 RSA, DSA (etc) code to select registered ENGINEs, cache defaults and
 functional references (etc), will generate trace summaries.
 
-=item C<OSSL_TRACE_CATEGORY_ENGINE_REF_COUNT>
+=item B<OSSL_TRACE_CATEGORY_ENGINE_REF_COUNT>
 
 Tracds the ENGINE reference counting.
```

Comment:
when you're at it, can you please fix this `Tracds` typo?

Reply:
Sure